### ISSUE TITLE
Rebase this to be an actual fork of CC:T

### ISSUE THREAD

**Issue by SkyTheCodeMaster** [TheMindVirus/CC-Modded#5]
This should be an actual fork of CC:T, instead of copy/pasted code.
That way if new features are added in CC:T, it can be easily added here.

Also CC:T is CCPL licenced, not MIT, and it is illegal to relicence it as anything other than CCPL.

**Comment by TheMindVirus** [comment]
~~Bump~~ Deliberate Duplicate of https://github.com/TheMindVirus/CC-Modded/issues/4
Now look what you've done, there's one that's open and one that's closed. Have it your way.

**Comment by SkyTheCodeMaster** [comment]
Yeah well you closed it without second thought. Having this be a copy/paste of CC:T instead of doing it the right way is downright idiotic and insane.
Also, none of your "changes" seem particularly useful. If they were so useful and required, why not PR them into CC:T?

**Comment by Merith-TK** [comment]
> Also, none of your "changes" seem particularly useful. If they were so useful and required, why not PR them into CC:T?

doing this would also allow these changes to be ported to the fabric version,

**Comment by TheMindVirus** [comment]
I did it the right way as well and no actual development took place at all, you didn't notice?

**Comment by Merith-TK** [comment]
The right way would've been clicking the fork button, and not changing the liscence, the rest of it is perfectly fine aside from the annoyance of you remove the git ignore rather than editing it for your personal uses

**Comment by TheMindVirus** [comment]
I'm not sure what you're referring to now, the license is the same as it was in both the "actual" fork and the custom fork, there are no changes there. Most corporate forks are GPL'd and for my repo's I go with MIT. In some way, the repository as a collective whole falls under multiple conflicting licenses, so I'm afraid I don't see what point it is you're trying to hammer home.

**Comment by Ocawesome101** [comment]
you have ~this repository and~ the curseforge page marked as being under the MIT license when in fact this repository uses the CCPL.  it's a small but important detail and should be changed - notice the CC:Tweaked CurseForge page uses a "custom license" (the CCPL).
